content/games/factions/rules.md

```diff
@@ -246,6 +246,23 @@ Well-Armed (1)
 |  Counter   |  8+   |  18”  |        Group        |                 -                  | End any Spell effect on the target as if its duration ended.                    |
 
+#### Taunts
+
+| Taunt                          | Check | Requirements                                     | Effect                                                                                                                                 |
+| :----------------------------- | :---: | :----------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------------------- |
+| Come and Die!                  |  6+   | -                                                | Cannot Move or Retreat (-1 FS if they would otherwise do so) and all enemy Groups are Reckless against them.                           |
+| We Are Unbreakable!            |  6+   | -                                                | Roll 3d6 and take the highest when testing Resolve but on failure, immediately Rout.                                                   |
+| We Leave No Friend Behind!     |  6+   | Must be within 6" of friendly Group              | While within 6" of a friendly Group re-roll any dice but lose 1d6 FS whenever a friendly Group within 6" Routs                         |
+| We Never Miss a Shot!          |  6+   | Groups with an MI profile only.                  | Must Shoot at the furthest enemy Group in range (if any) but may re-roll any misses when Shooting.                                     |
+| We'll Never Taste Your Blades! |  6+   | -                                                | Gain +1 T and +1 on to-hit rolls when Attacking but always count hits against them twice.                                              |
+| You'll Die with Us!            |  6+   | Must be Bloodied                                 | Inflict one hit for each taken when being Attacked but -1 T.                                                                           |
+| There Can Be Only One!         |  7+   | -                                                | Pick an enemy Group; when Attacking them gain +1 T and treat them as half as far away but must activate to Attack or Move toward them. |
+| We'll Fight in the Shade!      |  7+   | -                                                | Gain +1 T when an enemy Shoots them but gain Reckless trait against enemy Groups with an MI profile.                                   |
+| You'll Never Take Us Alive!    |  7+   | -                                                | Cannot Retreat (-1 FS if they would otherwise do so) but +1 on to-hit rolls in when Attacking or being Attacked.                       |
+| For the Captain!               |  8+   | Must be within 12" of Captain                    | While within 12" of Captain re-roll all failed Activations; otherwise re-roll all successful Activations and Resolve tests.            |
```

Comment:
```suggestion
| For the Captain!               |  8+   | Must be within 12" of Captain.                    | While within 12" of Captain, this Group rerolls all failed Activations. Otherwise, reroll all successful activations and Resolve tests.            |
```